nipype/interfaces/fsl/base.py

```diff
@@ -157,6 +158,48 @@ class FSLCommand(CommandLine):
     input_spec = FSLCommandInputSpec
     _output_type = None
 
+    references_ = [{'entry': BibTeX('@article{WoolrichJbabdiPatenaudeChappellMakniBehrens'
```

Comment:
I'd say we should place only the Neuroimage 2012 reference at the top level, and then have the specific references with the corresponding interfaces.